src/log-errors.js

```diff
@@ -3,9 +3,11 @@
 const ImmutableDBNotDefinedError = () => new Error('ImmutableDB instance not defined')
 const LogNotDefinedError = () => new Error('Log instance not defined')
 const NotALogError = () => new Error('Given argument is not an instance of Log')
+const NeedToUpgrade = () => new Error('OrbitDB: append is now disabled in 0.19. Please upgrade to the newest version of OrbitDB')
```

Comment:
Perhaps this could be improved by explicitly stating the versions: "append() is disabled as of 0.19.10. Please upgrade to 0.20.0 or newer."? Fine leaving it as it is, too.

Reply:
Easy enough to update! No problem